### DISCUSSION TITLE
zhenxun_bot日志一直报错无法连接到数据库,使用pgadmin可以正常连

### DISCUSSION THREAD

**Discussion by xiaotian1339** [SinKy-Yan/zhenxunbot-docker#17] [Q&A] [answered]
zhenxun_bot日志一直报错无法连接到数据库,使用pgadmin可以正常连

**Comment by SinKy-Yan** [comment, marked as answer]
你是在什么系统的docker部署的，好像在OpenWrt会有这个问题

**Reply by xiaotian1339** [reply]
解决了,在docker-compose配置中更改了数据库原始用户名和密码,但是zhenxun-bot容器里的配置还是默认的用户名和密码. 想当然的当成全局设置了.感谢
